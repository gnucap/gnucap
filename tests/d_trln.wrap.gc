
verilog

# "Qucsator style" TLIN
module TLIN (a, b)
	// ground \0
  parameter Z=50
  parameter L=100m
  parameter Alpha=1
  parameter Temp=26.85

  // local
  parameter c0=299792458.0
  tline #(.z(Z), .td(1./c0), .len(L) t(a, \0 , b, \0 );
endmodule

TLIN #(.Alpha(0),.L(100m),.Temp(26.85),.Z(50)) Line1 (.a(_net0),.b(out1));
TLIN #(.Alpha(0),.L(60m),.Temp(26.85),.Z(50)) Line1 (.a(_net0),.b(_net2));
TLIN #(.Alpha(0),.L(40m),.Temp(26.85),.Z(50)) Line1 (.a(_net2),.b(out2));

spice
.options noinsensitive
Vac _net0 0 ac 1
R1 out1 0 80
R1 out2 0 80

.print ac v(out*) vp(out*)
.ac 1e5 1e9 * 10
