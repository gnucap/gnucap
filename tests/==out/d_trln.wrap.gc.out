parameter Z=50 
          ^ ? missing type assuming real
parameter L=100m 
          ^ ? missing type assuming real
parameter Alpha=1 
          ^ ? missing type assuming real
parameter Temp=26.85 
          ^ ? missing type assuming real
parameter c0=299792458.0 
          ^ ? missing type assuming real
#Freq       v(out1)    v(out2)    vp(out1)   vp(out2)  
 100.K      1.         1.        -0.0075052 -0.0075052 
 1.Meg      1.         1.        -0.075052  -0.075052  
 10.Meg     1.0001     1.0001    -0.75059   -0.75059   
 100.Meg    1.0135     1.0135    -7.5726    -7.5726    
 1.G        1.3561     1.3561    -132.83    -132.83    
